javascript/code-challenges/graph/graph.js

```diff
@@ -0,0 +1,41 @@
+class Graph {
+  constructor() {
+    this.nodes = new Map();
+  }
+
+  addNode(value) {
+    if (!this.nodes.has(value)) {
+      this.nodes.set(value, []);
+    }
+    return value;
+  }
+
+  addEdge(node1, node2, weight = 1) {
+    if (!this.nodes.has(node1) || !this.nodes.has(node2)) {
+      throw new Error('Both nodes should already be in the graph');
+    }
```

Comment:
Should be an `&&`. We don't want it to make an edge if only one vertex is in the graph.